src/Microsoft.TemplateEngine.Cli/TemplateInvoker.cs

```diff
@@ -55,10 +62,10 @@ internal async Task<CreationResultStatus> InvokeTemplate(ITemplateMatchInfo temp
                 auth = TelemetryHelper.HashWithNormalizedCasing(TelemetryHelper.GetCanonicalValueForChoiceParamOrDefault(templateToInvoke.Info, "auth", inputAuthValue));
             }
 
-            bool argsError = CheckForArgsError(templateToInvoke, out string commandParseFailureMessage);
+            bool argsError = CheckForArgsError(templateToInvoke, out string? commandParseFailureMessage);
             if (argsError)
             {
-                _telemetryLogger.TrackEvent(_commandName + TelemetryConstants.CreateEventSuffix, new Dictionary<string, string>
+                _telemetryLogger.TrackEvent(_commandName + TelemetryConstants.CreateEventSuffix, new Dictionary<string, string?>
```

Comment:
nit: Wondering if it would make more sense to do `?? "<unknown?"` for nullable strings in this dictionary

Reply:
not sure, looking on the examples:
- language tend to be set for most of templates, exception can be for example `gitignore`
- `framework` and especially `auth` is not applicable for most of templates, therefore not sure if `<unknown>` is not confusing.

Reply:
I agree, ignore this comment, I for some reason though `null` means something went bad...